### ISSUE TITLE
QA Leo P3A metrics for Android

### ISSUE THREAD

**Issue by DJAndries** [brave/brave-browser#36323]
The following metrics require testing on Android, with the upcoming release of Leo.

`Brave.AIChat.Enabled.2`
`Brave.AIChat.UsageDaily.2`
`Brave.AIChat.UsageMonthly`
`Brave.AIChat.ChatCount`
`Brave.AIChat.AvgPromptCount`
`Brave.AIChat.UsageWeekly`
`Brave.AIChat.NewUserReturning`
`Brave.AIChat.LastUsageTime`

Please refer to test plan in https://github.com/brave/brave-core/pull/19569.

**Comment by DJAndries** [comment]
cc @GeetaSarvadnya

**Comment by kjozwiak** [comment]
Closing as the above has been in `Release` for some time. Looks like https://github.com/brave/brave-core/pull/19569 landed in `1.59.x` and the original issue was labelled as `QA/No` via https://github.com/brave/brave-browser/issues/32050 as `Leo` wasn't ready at the time.

**Comment by hffvld** [comment]
Verified on `Pixel 7` using version(s):
```
Device/OS: Pixel 7 / panther_beta-user 14 AP21.240119.009 release-keys
Brave build: 1.63.164
Chromium: 122.0.6261.69 (Official Build) (64-bit) 
```

Filed follow-up issue #36410, #36439
---------------------------

**STEPS:**
1. Follow the steps from https://github.com/brave/brave-core/pull/19569#issue-1835805432
2. *Verify*

---------------------------

*Start with a `Fresh profile`, ensure `none` of the metrics exist*

|1|2|
|:-:|:-:|
| ![1](https://github.com/brave/brave-browser/assets/128532965/5f59e6f9-f5c8-4536-b5d4-d8880e3e183e) | ![2](https://github.com/brave/brave-browser/assets/128532965/6a93ef3b-f1c3-4be8-bdf6-23c252173cf1) |

*Start a chat with `Leo`, send only `one` prompt, and ensure the enabled metric `Brave.AIChat.Enabled.2` is set to `1`, the usage daily metric `Brave.AIChat.UsageDaily.2` is set to `1`, the chat count `Brave.AIChat.ChatCount` is set to `0`, and the average prompt count metric `Brave.AIChat.AvgPromptCount` is set to `0`*

|1|2|3|4|
|:-:|:-:|:-:|:-:|
| ![1](https://github.com/brave/brave-browser/assets/128532965/da3bae9a-41bf-4719-9a2a-93120ff6ea43) | ![2](https://github.com/brave/brave-browser/assets/128532965/09a29207-19b3-454a-b819-d5510a9f6f37) | ![3](https://github.com/brave/brave-browser/assets/128532965/a6288262-e8e5-4d54-bbbb-148d8b099d0b) | ![4](https://github.com/brave/brave-browser/assets/128532965/a0909b6f-fcc3-42aa-9b06-54419d228f03) |

*Ensure usage daily metric `Brave.AIChat.UsageDaily.2` is `sent`, close the browser, advance system time by `one day` and start the browser. Ensure the usage daily metric `Brave.AIChat.UsageDaily.2` is `gone`, but all other metrics are present*

|1|2|3|4|
|:-:|:-:|:-:|:-:|
| ![1](https://github.com/brave/brave-browser/assets/128532965/38f65d07-4886-49e1-b2a5-fbdce242a864) | ![2](https://github.com/brave/brave-browser/assets/128532965/a5040457-ae0d-47d8-948d-68bcae93034f) | ![3](https://github.com/brave/brave-browser/assets/128532965/94046f2f-d868-4c0d-b284-81ade572061e) | ![4](https://github.com/brave/brave-browser/assets/128532965/797de7ee-e03b-4c2b-8404-324e965fda25) |

*Start another chat with a single prompt, ensure the chat count metric `Brave.AIChat.ChatCount` is `1` and the average prompt count `Brave.AIChat.AvgPromptCount` is `0`. Ensure usage daily metric `Brave.AIChat.UsageDaily.2` is `1`*

|1|2|3|
|:-:|:-:|:-:|
| ![1](https://github.com/brave/brave-browser/assets/128532965/84652486-26f3-48ba-bcd8-72b4669c8c7f) | ![2](https://github.com/brave/brave-browser/assets/128532965/ea0fa8e6-32fe-4433-b870-7689af257465) | ![3](https://github.com/brave/brave-browser/assets/128532965/8b9b1024-84ea-4ebb-bf4e-37cd839645c4) |

*Start another chat and send a total of `7 prompts` in the chat. Ensure the chat count metric `Brave.AIChat.ChatCount` is still `1`, but the average prompt count metric `Brave.AIChat.AvgPromptCount` is `1`*

|1|2|3|
|:-:|:-:|:-:|
| ![1](https://github.com/brave/brave-browser/assets/128532965/9ab5fff8-3e27-4f0c-af38-10d5adc22e3c) | ![2](https://github.com/brave/brave-browser/assets/128532965/8dc77efe-6894-4d90-b94d-9a127e9c62a9) | ![3](https://github.com/brave/brave-browser/assets/128532965/0043ec51-4cab-44f8-b231-1ea62d83658b) |

*Close the browser, advance the system time by `7 days` and ensure that the chat count `Brave.AIChat.ChatCount` and average prompt count `Brave.AIChat.AvgPromptCount` metrics are gone*

|1|2|3|4|
|:-:|:-:|:-:|:-:|
| ![1](https://github.com/brave/brave-browser/assets/128532965/f311cab0-10b8-4536-aa1a-fed6cc879371) | ![2](https://github.com/brave/brave-browser/assets/128532965/5865055f-6d3b-40da-9903-0fd50cb2ceb5) | ![3](https://github.com/brave/brave-browser/assets/128532965/7eddf472-0ef1-40dc-bca0-ec4283d05a78) | ![4](https://github.com/brave/brave-browser/assets/128532965/791e460e-3aa0-4bfb-9477-9e6ffef0ca0a) |
